Procfile

```diff
@@ -48,3 +48,15 @@ jobs: bundle exec rails runner bin/threaded.rb
 #dj8: bundle exec script/delayed_job -i 8 run
 #dj9: bundle exec script/delayed_job -i 9 run
 #dj10: bundle exec script/delayed_job -i 10 run
+
+###################
```

Comment:
Thanks! I think only Heroku and Docker users will need to configure it. When redis is installed on the same host as Huginn the default URL will work. I added a note to your updated version.